### DISCUSSION TITLE
How do I make a mode transition without ESC?

### DISCUSSION THREAD

**Discussion by akhasano** [pit-ray/win-vind#88] [General]
Hello,
It is not working for me https://pit-ray.github.io/win-vind/faq/

I would change ESC to any other button (ideally LCtrl+Win instead ESC) but it doesn't work. 
Does it should work in any Windows? I have Windows 10 Pro 21H2 and it doesn't work.
What I am do, I am using portable version unpacked to "D:\soft\win-vind":
* Run as regular user
* From tray - open root directory
* Open .vindrc and put text from page: https://pit-ray.github.io/win-vind/faq/

switch to gui normal, ":e source", trying to restart app.
Expectation: <caps>+left or right or down works as <esc>+left or right or down. But it still works with "esc" and ignores "caps".
If i put "set cmd_fontname = Times New Roman" and restart - I can see font is changed from Verdana to TNR.
Version: 4.3.3

Can you please advice? 
Thanks.

**Comment by pit-ray** [comment]
Sorry for the late reply.

I have confirmed that it does not work with the FAQ settings. Perhaps there was a bug in partial mapping due to the macro feature introduced in v4.3.
However, full-mapping or function-id-definition are avalilable. It will be a little duplicative and more long, but it will work.

- **Full mapping**
```vim
inoremap <lctrl-win> <esc-left>
```

- **Function ID definition**
```vim
inoremap <lctrl-win> to_gui_normal
```
In this case, you can also disable ESC by using unmap as follows
```vim
iunmap <esc-left>
```

The same settings can be made in other modes.

Thanks for using it.

**Comment by pit-ray** [comment]
I have posted this bug in the issue.

https://github.com/pit-ray/win-vind/issues/93
